### ISSUE TITLE
Crowd-source tips

### ISSUE THREAD

**Issue by brittanydionigi** [brittanystoroz/devtip-of-the-day#2]
We need to take recommendations of what tooling to support, and we need help manually adding all the tips and tricks into our dataset for each of those tools.

**Comment by shakeelmohamed** [comment]
Can you take recommendations via PR? If so, you can setup a simple CI check to validate `tips.json`

**Comment by brittanydionigi** [comment]
@shakeelmohamed Yes, PRs are welcome! Though this isn't exactly what I mean by crowd-source tips. The extension is a tip-of-the-day addon for developers to learn new keyboard shortcuts and workflow enhancements. I need those kinds of tips. Like the measly 4 I have right now in `tips.json` ;) You're welcome to file an issue and PR for integrating things like CI and other overall enhancements for the addon.
